essentials/src/chain_head_subscription.rs

```diff
@@ -103,7 +103,7 @@ impl ChainHeadSubscription {
 	// Per consumer
 	async fn run_per_node(
 		mut update_channel: Sender<ChainHeadEvent>,
-		url: String,
+		url: String, // `String` rather than `&str` because we spawn this method as an asynchronous task
```

Comment:
@sandreim but we still have to use strings for methods we run as a task, like that